src/strapi_kit/export/jsonl_reader.py

```
@@ -0,0 +1,195 @@
+            return self._metadata
+
+        line = self._file.readline()
+        self._current_line = 1
+
+        if not line:
+            raise FormatError("Empty JSONL file")
+
+        try:
+            record = json.loads(line)
+        except json.JSONDecodeError as e:
+            raise FormatError(f"Invalid JSON on line 1: {e}") from e
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)